components/atoms/Counter/index.jsx

```diff
@@ -0,0 +1,11 @@
+import styles from './styles.module.scss';
+
+const Counter = ({ quantity = 0 }) => {
+	return (
+		<>
+			<p className={styles.counter}>{quantity}</p>
+		</>
+	);
+};
```

Comment:
podes ahorrarte los parentesis y los <></> fragments si solo vas a retornar un parrafo.
```suggestion
const Counter = ({ quantity = 0 }) => {
	return <p className={styles.counter}>{quantity}</p>
};
```